### DISCUSSION TITLE
Glider configurable url prefix

### DISCUSSION THREAD

**Discussion by DanielSpravtsev** [awcodes/filament-curator#337] [Ideas]
Hey, @awcodes! 
Is there a way we can move glider ["/curator/" signed url prefix](https://github.com/awcodes/filament-curator/blob/153b0f466083eabdde18cbfa7daf8bd85a31600b/src/View/Components/Glider.php#L103C62-L103C62) to curator config, glider section?

**Comment by awcodes** [comment]
Hm, not sure how I feel about using a config value in a routes file. What's the use case?

**Reply by DanielSpravtsev** [reply]
Use case is simple: just wanted more clear url path (site.com/media/*) 😄

**Reply by awcodes** [reply]
you know it's a proxy url though right, and not a direct path to the files?

**Reply by DanielSpravtsev** [reply]
Of course. I think in my case it'll be better to just use own view component to render .img with needed prefix.

**Reply by awcodes** [reply]
Are you trying to make your own 'glider' component? or just use the URL inline?

You could always make you're own proxy route too.

**Reply by DanielSpravtsev** [reply]
> Are you trying to make your own 'glider' component? or just use the URL inline?

Not trying, just asked is it possible to use own proxy route prefix. 
I can extend from your Glider component and just redefine methods with hardcoded prefixes

> You could always make you're own proxy route too.

Without switching to own component implementation? How?

**Reply by awcodes** [reply]
just copy the functionality of the 'curator' route to your own route in `web.php` :)

I'm just not sold on it being configurable since seeing the name 'curator' in a route list helps to identify what it is.
